ios/Demo-iOS/Sources/Views/AppRootView.swift

```diff
@@ -86,21 +86,6 @@ struct AppRootView: View {
                     .setLogLevel(.debug)
                     .setEnableNetworkLogging(true)
                     .build()
-
-                if let baseURL = URL(string: config.siteApiRoot) {
-                    let service = EditorService(
-                        siteURL: config.siteUrl,
-                        networkSession: URLSession.shared
-                    )
-
-                    do {
-                        try await service.setup(&updatedConfiguration)
```

Comment:
This is done by `EditorViewController`: https://github.com/wordpress-mobile/GutenbergKit/blob/109fc5f453c16550801239a3a5aecc91e663a9b3/ios/Sources/GutenbergKit/Sources/EditorViewController.swift#L257.